src/Utils.php

```diff
@@ -181,4 +181,26 @@ function ($a, $b) {
             }
         );
     }
+
+    /**
+     * Join names in natural language wrapped in backticks, e.g. `a`, `b` and `c`.
+     *
+     * @param string[] $names
+     *
+     * @return string
+     */
+    public static function naturalLanguageJoinWithBackticks(array $names)
```

Comment:
empty array is invalid input

Reply:
what do you mean? do you want exception to be thrown for empty array?

Reply:
IMO it's fine, just like `implode` does not trigger any error when an empty array is used.

Reply:
we won't call it for empty array, right?
yet, currently there is a boilerplate code to support handling it. and with it, we shall check if `output !== ''` whenever we want to use this method